services/autoscaler-adapter/claudie_provider/claudie_provider_test.go

```diff
@@ -23,7 +23,7 @@ func TestRefresh(t *testing.T) {
 
 	defer func() {
 		if err := cc.Close(); err != nil {
-			log.Err(err).Msgf("Failed to close adapter connection")
+			log.Error().Err(err).Msgf("Failed to close adapter connection")
```

Comment:
```
Err starts a new message with error level with err as a field if not nil or with info level if err is nil.

You must call Msg on the returned event in order to send the event. 
```
I don't understand this change. From docs, the `Err()` already sets level of the message.